ui/e2e/pages/PoolPage.js

```diff
@@ -0,0 +1,144 @@
+import { DEX_TARGET } from "../config";
+import { tokenSelection } from "./components/TokenSelection";
+import { GenericPage } from "./GenericPage";
+
+export class PoolPage extends GenericPage {
+  constructor() {
+    super();
+    this.el = {
+      detailsPriceMessage: "[data-handle='details-price-message']",
+      detailsMinimumReceived: "[data-handle='details-minimum-received']",
+      detailsPriceImpact: "[data-handle='details-price-impact']",
+      detailsLiquidityProviderFee:
+        "[data-handle='details-liquidity-provider-fee']",
+      addLiquidityButton: '[data-handle="add-liquidity-button"]',
+      actionsButton: '[data-handle="actions-go"]',
+      poolPricesForwardNumber: '[data-handle="pool-prices-forward-number"]',
+      poolPricesForwardSymbols: '[data-handle="pool-prices-forward-symbols"]',
+      poolPricesBackwardNumber: '[data-handle="pool-prices-backward-number"]',
+      poolPricesBackwardSymbols: '[data-handle="pool-prices-backward-symbols"]',
+      poolEstimatesForwardNumber:
+        '[data-handle="pool-estimates-forwards-number"]',
+      poolEstimatesForwardSymbols:
+        '[data-handle="pool-estimates-forwards-symbols"]',
+      poolEstimatesBackwardNumber:
+        '[data-handle="pool-estimates-backwards-number"]',
+      poolEstimatesBackwardSymbols:
+        '[data-handle="pool-estimates-backwards-symbols"]',
+      poolEstimatesShareNumber: '[data-handle="pool-estimates-share-number"]',
+      managePoolButton: (tokenA, tokenB) =>
+        `[data-handle="${tokenA}-${tokenB}-pool-list-item"]`,
+      totalPooled: (token) => `[data-handle="total-pooled-${token}"]`,
+      totalPoolShare: '[data-handle="total-pool-share"]',
```

Comment:
I really love the way you have consolidated our selectors like this. I am not crash hot on having pages abstracted away but I can live with it as it has been done in a way that is pretty clear however to me selector consolidation is the main value of this refactoring.